app/controllers/groups_controller.rb

```diff
@@ -30,6 +30,7 @@ def new
     assignment = Assignment.find(params[:assignment_id])
     begin
       assignment.add_group(params[:new_group_name])
+      flash.now[:success] = I18n.t('groups.rename_group.success')
```

Comment:
Please use the helpers `flash_now` and `flash_message` instead of accessing the flash hashes directly.